aloscene/mask.py

```diff
@@ -110,7 +97,56 @@ def __get_view__(self, title=None):
             )
         return View(frame, title=title)
 
+    def masks2panoptic(self):
```

Comment:
Where is this method used ?

Reply:
@thibo73800  In alonet.callbacks.object_detector_callback.py, on `log_masks`  function. It is a function like a `rgb2id`, but adapted to aloscene.Mask